### PR TITLE
fix walkthrough example

### LINKED ISSUE #378: [walkthrough] ServiceMonitor does not work
I followed the [walkthrough](https://github.com/kubernetes-sigs/prometheus-adapter/blob/master/docs/walkthrough.md), but the service monitor port name does not match the port name of the actual service created with:
```
$ kubectl create service clusterip sample-app --tcp=80:8080
```
The service monitor matches with a port named http, but for me, the command above created a port name "80-8080" instead of "http".
I'm not sure if this is something that changed at some point with kubeclt binary version. My version:
```
kubectl version
Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.4", GitCommit:"e87da0bd6e03ec3fea7933c4b5263d151aafd07c", GitTreeState:"clean", BuildDate:"2021-02-21T20:23:45Z", GoVersion:"go1.15.8", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.6", GitCommit:"d32e40e20d167e103faf894261614c5b45c44198", GitTreeState:"clean", BuildDate:"2020-05-20T13:08:34Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
```

I'll happily provide a PR with a doc fix, but wanted to make sure this does not break the walkthrough for users with older versions of kubectl.

### PR DESCRIPTION
**Pull request by TheKangaroo**
fixes #378 

This PR makes sure, the ServiceMonitor and the Service port name match each other.

### CODE REVIEW

**Comment by TheKangaroo** [comment]
cla signed

File: docs/walkthrough.md [hunk 1]
```diff
@@ -94,9 +94,31 @@ spec:
...
+  - name: http
+    port: 80
+    protocol: TCP
+    targetPort: 8080
+  selector:
+    app: sample-app
+  type: ClusterIP
+```
+
+</details>
```
Comment (dgrisonnet) [inline docs/walkthrough.md:117]: The opening `<detail>` block is missing above, hence the `sample-app.service.yaml` content isn't collapsible.

**Review by dgrisonnet — COMMENTED** [review]
Hello @TheKangaroo, thank you for fixing this part of the documentation! I just have one small nit comment, but it looks good to me overall :slightly_smiling_face: 

/ok-to-test
/check-cla

File: docs/walkthrough.md (hunk 1 above)
Comment (TheKangaroo) [reply docs/walkthrough.md:119]: good catch. fixed it :)

**Pull request merged**